ostree-ng.sh

```diff
@@ -406,10 +431,18 @@ greenprint "🧹 Clean up compose"
 sudo composer-cli compose delete "${COMPOSE_ID}" > /dev/null
 sudo composer-cli blueprints delete container > /dev/null
 
-# Remove tag from quay.io repo
 greenprint "Remove tag from quay.io repo"
 skopeo delete --creds "${QUAY_USERNAME}:${QUAY_PASSWORD}" "${QUAY_REPO_URL}:${QUAY_REPO_TAG}"
 
+# # Remove tag from quay.io repo
```

Comment:
Let's remove the comment out code if we do not need them. Thanks.

Reply:
Ok, done! Thanks!